docs/docs/extra-features.md

```diff
@@ -5,14 +5,15 @@
 The following images types are supported in sku:
 `bmp`, `gif`, `jpeg`, `png`, `svg`, `webp` and `avif`.
 
-> Browser support for `webp` and `avif` varies. To ensure compatability across browsers, consider providing fallback image formats using the [`picture`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture) element.
->
-> ```html
-> <picture>
->   <source srcset="image.webp" type="image/webp" />
->   <img src="image.png" />
-> </picture>
-> ```
+?> Browser support for `webp` and `avif` varies. To ensure compatability across browsers, consider providing fallback image formats using the [`picture`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture) element.
```

Comment:
What does this look like on the docs site? Is this valid markdown?

Reply:
![Screenshot 2024-12-04 at 10 04 37 AM](https://github.com/user-attachments/assets/1aa93fa9-b366-44c4-be5d-f14024e685b2)

(Link)[https://docsify.js.org/#/helpers?id=general-tips]
> docsify extends Markdown syntax to make your documents more readable.

Not valid vanilla markdown, but valid in the context of Docsify

Reply:
Oh, had no idea that docsify supported that! Awesome.